### PR TITLE
VAN-130: Add translation strings

### PR DESCRIPTION
**Pull request by zainab-amir**
Added missing translation strings for Login and Register Page.

Ticket: https://openedx.atlassian.net/browse/VAN-130

### CODE REVIEW

File: src/logistration/RegistrationPage.jsx [hunk 1]
```diff
@@ -153,7 +153,7 @@ class RegistrationPage extends React.Component {
 
   renderCountryList() {
     const locale = getLocale();
```
Comment (adeelkhan) [inline src/logistration/RegistrationPage.jsx:155]: This code is removed in my PR. We dont need this as the backend is giving us country list. https://github.com/edx/frontend-app-logistration/pull/31/files#diff-e871e903f58eef8e0d9182a0de24715ead7eb4cb866505c91e1ac1f329811793L154

File: src/logistration/RegistrationPage.jsx [hunk 2]
```diff
@@ -249,9 +253,11 @@ class RegistrationPage extends React.Component {
             <ValidationFormGroup
               for="email"
               invalid={this.state.errors.email !== ''}
-              invalidMessage="Enter a valid email address that contains at least 3 characters."
+              invalidMessage={intl.formatMessage(messages['logistration.email.validation.message'])}
```
Comment (adeelkhan) [inline src/logistration/RegistrationPage.jsx:256]: Secondly a lot of changes are done in register form fields in another PR https://github.com/edx/frontend-app-logistration/pull/43 .Error messages are resolved realtime via backend and gets injected from state variables. So i don't think we can add translation like this. So error string would get translated via backend.

**Comment by zainab-amir** [comment]
@adeelkhan I will wait for your PR to be merged and will rebase your changes. Once that is done, all unwanted code will be removed

**Comment by adeelkhan** [comment]
@zainab-amir I am sorry for a lot of extra effort. Thanks!

**Review by waheedahmed — APPROVED** [review]

**Review by adeelkhan — APPROVED** [review]
